ProcessLib/TwoPhaseFlowWithPP/TwoPhaseFlowWithPPLocalAssembler.h

```diff
@@ -20,13 +20,27 @@
 #include "ProcessLib/Parameter/Parameter.h"
 #include "ProcessLib/Utils/InitShapeMatrices.h"
 
-#include "MaterialLib/TwoPhaseModels/TwoPhaseFlowWithPPMaterialProperties.h"
+#include "TwoPhaseFlowWithPPMaterialProperties.h"
 #include "TwoPhaseFlowWithPPProcessData.h"
 
 namespace ProcessLib
 {
 namespace TwoPhaseFlowWithPP
 {
+template <typename NodalMatrixType>
+struct IntegrationPointData final
+{
+    explicit IntegrationPointData(TwoPhaseFlowWithPPMaterialProperties&
+            material_property_)
+        : mat_property(material_property_)
+    {
+    }
+    TwoPhaseFlowWithPPMaterialProperties&
+        mat_property;
+
+    double integration_weight;
+    NodalMatrixType massOperator;
```

Comment:
The integration weight and the mass operator do not change during the simulation, do they?
If not, make them const here (need to pass them through constructor).